packages/core/src/browser/style/dialog.css

```diff
@@ -6,17 +6,17 @@
  */
 
 .p-Widget.dialogOverlay {
+    z-index: 5000;
     display: flex;
     flex-direction: column;
+    position: fixed;
     width: 100vw;
     height: 100vh;
     top: 0;
     left: 0;
-    display: flex;
```

Comment:
Could you explain why you removed `flex` here?

Reply:
It's already defined a few lines above.